gameplan.txt

```diff
@@ -0,0 +1,50 @@
+gameplan
```

Comment:
This file wasn't meant to be here. The auto-staging feature in Git Kraken when switching branches is really annoying...